test/browser_tests/spec_suites/shared/shared_contact-us.js

```diff
@@ -29,7 +30,7 @@ describe( 'Contact Us Page', function() {
     expect( secondPhone.getAttribute( 'href' ) ).toBe( 'tel:8557292372' );
   } );
 
-// This hasn't been released to beta yet and should fail when testing that url
+  // TODO: This hasn't been released to beta yet and should fail when testing that url.
```

Comment:
This todo note can be removed entirely, this has been on beta and the test has been running for some time now.